vespalib/src/vespa/vespalib/stllike/cache.h

```diff
@@ -127,6 +139,10 @@ class cache {
         //   size_bytes() <= capacity() && size() <= maxElements()
         using Lru::trim;
 
+        // Invokes functor `fn` for each segment key in LRU order (new to old)
```

Comment:
technically the wrong order. Maybe we should consider enforcing a big enough sketch to not decay due to filling in all cache elements whenever a size > 0 is selected?

Reply:
Added a lower bound on the existing cache `size()` when setting a non-zero sketch size. The iteration order is an artefact of the only available iteration order of the underlying `lrucache_map`, so not doing anything with it for now at least.